From 728bdc6a2e922b70a94c81a28d681fa56f41210e Mon Sep 17 00:00:00 2001
From: Nick <nick@khadas.com>
Date: Wed, 23 Jan 2019 11:40:43 +0800
Subject: [PATCH] Bluetooth: Fix spurious error message

This message was debug message before 2064ee332e4c1b7495cf68b. Looks
like it was changed by accident in that patch. This is causing some
error messages when doing BR/EDR discovery since Inquiry Command
generates Command Status event, not Command Complete.

I'm sending this as RFC for now since while things seem to work fine
despite this condition (and it was like that for quite some time) it
feels like this is indeed case that should be handled by HCI request
code.
---
 net/bluetooth/hci_event.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/net/bluetooth/hci_event.c b/net/bluetooth/hci_event.c
index ef9928d..a620745 100644
--- a/net/bluetooth/hci_event.c
+++ b/net/bluetooth/hci_event.c
@@ -5712,8 +5712,7 @@ static bool hci_get_cmd_complete(struct hci_dev *hdev, u16 opcode,
 	}
 
 	if (hdr->evt != HCI_EV_CMD_COMPLETE) {
-		bt_dev_err(hdev, "last event is not cmd complete (0x%2.2x)",
-			   hdr->evt);
+		BT_DBG("last event is not cmd complete (0x%2.2x)", hdr->evt);
 		return false;
 	}
 
-- 
1.9.1

